relay-general/src/store/transactions/rules.rs

```diff
@@ -176,25 +161,18 @@ pub struct TransactionNameRule {
     pub pattern: LazyGlob,
     /// Date time when the rule expires and it should not be applied anymore.
     pub expiry: DateTime<Utc>,
-    /// Object containing transaction attributes the rules must only be applied to.
-    #[serde(default)]
-    pub scope: TransactionNameRuleScope,
```

Comment:
The condition for applying clustering rules has changed from `source == "url"` to `source in ["url", "sanitized", null]`. Instead of extending the config protocol to support lists, I suggest we drop this field from the config and let Relay decide when to apply these rules.

Reply:
Note that old Relays will still deserialize the missing scope as `{"source": "url"}` because of the `#[serde(default)]` attribute, so this change is backward compatible.